zerver/data_import/mattermost.py

```diff
@@ -262,7 +290,7 @@ def process_raw_message_batch(realm_id: int,
                               subscriber_map: Dict[int, Set[int]],
                               user_id_mapper: IdMapper,
                               user_handler: UserHandler,
-                              get_recipient_id: Callable[[ZerverFieldsT], int],
+                              get_recipient_id_from_receiver_name: Callable[[str, int], int],
```

Comment:
Can the `get_recipient_id_from_receiver_name` refactor be split to a prep commit?

Reply:
Done.